### PR TITLE
fix(PG): RHICOMPL-3729 upgrade postgres to 14 in compose file

### PR DESCRIPTION
**Pull request by RoamingNoMaD**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -100,7 +100,7 @@ services:
     links:
       - kafka
   db:
-    image: docker.io/debezium/postgres
+    image: docker.io/debezium/postgres:14-alpine
```
Comment (Victoremepunto) [inline docker-compose.yml:103]: Would it make sense to  you try using the Clowder PostgreSQL 14 image ? 

quay.io/cloudservices/postgresql-rds:14-1

Comment (skateman) [reply docker-compose.yml:103]: As far as I understand, this is a debezium-flavored PG with some extras that we need in development :thinking:

Comment (skateman) [reply docker-compose.yml:103]:
```
This image is based upon [postgres:11-apline](https://hub.docker.com/_/postgres/) and adds two logical decoding plug-ins:

[postgres-decoderbufs](https://github.com/debezium/), based on Protocol Buffers and maintained by the Debezium community
[wal2json](https://github.com/eulerto/wal2json), based on JSON and maintained by the wal2json community
```

Comment (romanblanco) [reply docker-compose.yml:103]: https://github.com/RedHatInsights/ros-backend/pull/267#discussion_r1022626729

**Comment by skateman** [comment]
@RoamingNoMaD CI failure seems sporiadic, please rebase that will trigger a will re-run and probably pass

**Pull request merged**